marketplace/cloudinary-assets/src/components/sortable/sortable.tsx

```diff
@@ -95,12 +94,11 @@ export class SortableComponent extends React.Component<
     return (
       <SortableList
         onSortEnd={this.onSortEnd}
-        axis="x"
-        pressDelay={0}
+        axis="xy"
         resources={this.state.resources}
         config={this.props.config}
         deleteFnc={this.deleteItem}
-        useDragHandle
+        useDragHandle={true}
```

Comment:
Surprised this isn't a lint rule. Usually you could just use the shorthand version.
```suggestion
        useDragHandle
```